georocket-server/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume that grade behave like maven and use logback-test.xml only for unit tests. 

"If you are using Maven and if you place the logback-test.xml under the src/test/resources folder, Maven will ensure that it won't be included in the artifact produced. Thus, you can use a different configuration file, namely logback-test.xml during testing, and another file, namely, logback.xml, in production." 
Src: http://logback.qos.ch/manual/configuration.html